drivers/hetznercloud/destroy.go

```diff
@@ -28,10 +30,13 @@ func (p *provider) Destroy(ctx context.Context, instance *autoscaler.Instance) e
 
 	logger.Debugln("deleting instance")
 
-	_, err = p.client.Server.Delete(ctx, &hcloud.Server{ID: id})
+	msg, err := p.client.Server.Delete(ctx, &hcloud.Server{ID: id})
 
 	if err != nil {
-		if err.Error() == "hcloud: server responded with status code 404" {
+		// json response contains a code=not_found field
+		msgBytes, err2 := io.ReadAll(msg.Response.Body)
```

Comment:
err2 isn't very descriptive. can you rename it, please?

Reply:
Changed. Sorry for the delay.